src/main/java/com/dumbdogdiner/stickyapi/common/nbt/NbtJsonAdapter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.nbt;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import lombok.experimental.UtilityClass;
+import org.jetbrains.annotations.NotNull;
+
+import java.text.MessageFormat;
+
+/**
+ * Utility class for easy creation of the appropriate {@link NbtTag} from an existing {@link JsonElement}
+ */
+@UtilityClass
+public final class NbtJsonAdapter {
+    /**
+     * Creates the appropriate tag from a given {@link JsonElement}, but never creates a {@link NbtJsonTag}
+     * @param jse The incoming {@link JsonElement} to convert
+     * @return the new {@link NbtTag}
+     */
+    public static NbtTag jsonToNbt(@NotNull JsonElement jse) {
```

Comment:
```suggestion
    public static NbtTag fromJson(@NotNull JsonElement element) {
```
again personal preference, to make it look cleaner